### PR TITLE
Gh 783/uikit action details

### PR DESCRIPTION
**Pull request by DmitryBespalov**
This implements the functionality of the Action Details and MultiSend screens.

The UI adjustments so that it looks like in the Zeplin will follow in the next PR, as well as implementation of the multiSend transaction details screen.

### CODE REVIEW

File: Multisig/UI/App/UIKit/Transaction/ActionDetailAddressCell.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  ActionDetailAddressCell.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 05.01.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class ActionDetailAddressCell: ActionDetailTableViewCell {
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/UIKit/Transaction/ActionDetailAddressCell.swift:11]: ActionDetails?

**Comment by DmitryBespalov** [comment]
this needs more work

File: Multisig/UI/App/UIKit/Transaction/ActionDetailViewController.swift [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+                        let startIndex = self.pushIndex()
+
+                        for item in value {
+                            self.buildValue(item, nestingLevel: nestingLevel + 1)
+                        }
+
+                        let endIndex = self.popIndex()
+
+                        // remove added cells because thye
```
Comment (sche) [inline Multisig/UI/App/UIKit/Transaction/ActionDetailViewController.swift:162]: is it not finished comment?

**Review by sche — APPROVED** [review]
It seems that SwiftUI impl was cleaner, wdyt?

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request closed without merging**